app/controllers/posts_controller.rb

```diff
@@ -25,4 +25,11 @@ def create
       redirect_to new_user_post_path
     end
   end
+
```

Comment:
- [ ] Kindly add  CanCanCan for Post authorization, this is a project requirement

Reply:
I am also running into this error when I run your app:
![image](https://user-images.githubusercontent.com/61408860/193061220-3913fc83-0e6e-4714-bc1b-e43f9160b43c.png)

Reply:
@danmainah your request is not clear can you please elaborate on it? 

- I have added CanCanCan for authorization to delete a post and delete a comment. You can see it in the [ability.rb](https://github.com/darikmohammed/blog-app/pull/10/files#diff-0fdc2ce2595c4fe479d6a97b404013651e8a0e2d4b02d93c87843b3a30103d9f)

- I am using Tailwind CSS to style my application and it is working fine on my end.

I have shared the work of my project in the video below.

https://user-images.githubusercontent.com/56404835/193068869-29a2849c-fd24-4f44-8554-07f085e71706.mov